enslig-forsorger-test/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/TestSøknadBuilder.kt

```diff
@@ -0,0 +1,486 @@
+            return listOf(Aksjeselskap(Søknadsfelt("Navn", "Fima abc"),
+                    Søknadsfelt("arbeidsmengde", 50)))
+        }
+
+        private fun defaultUtdanning(): UnderUtdanning {
+            return UnderUtdanning(Søknadsfelt("Skole/utdanningssted", "UiO"),
+                    Søknadsfelt("Bakoverkompatibel Utdanning",
+                            TidligereUtdanning(Søknadsfelt("Linje/kurs/grad",
+                                    "Profesjonsstudium Informatikk"),
+                                    Søknadsfelt("Når skal du være elev/student?",
+                                            MånedÅrPeriode(Month.APRIL,
+                                                    2020,
+                                                    Month.JANUARY,
+                                                    2021)
+                                    ))),
+                    Søknadsfelt("Utdanning",
+                            GjeldendeUtdanning(Søknadsfelt("Linje/kurs/grad",
+                                    "Profesjonsstudium Informatikk"),
+                                    Søknadsfelt("Når skal du være elev/student?",
+                                            Datoperiode(LocalDate.of(1999, 1, 1),
+                                                    LocalDate.of(2004, 10, 12))
+                                    ))),
+                    Søknadsfelt("Er utdanningen offentlig eller privat?",
+                            "Offentlig"),
+                    Søknadsfelt("Heltid, eller deltid", "Deltid"),
+                    Søknadsfelt("Hvor mye skal du studere?", 300),
+                    Søknadsfelt("Hva er målet med utdanningen?",
+                            "Økonomisk selvstendighet"),
+                    Søknadsfelt("Har du tatt utdanning etter grunnskolen?", true),
+                    Søknadsfelt("Tidligere Utdanning",
+                            listOf(TidligereUtdanning(Søknadsfelt("Linje/kurs/grad",
+                                    "Master Fysikk"),
+                                    Søknadsfelt("Når var du elev/student?",
+                                            MånedÅrPeriode(Month.JANUARY,
+                                                    1999,
+                                                    Month.OCTOBER,
+                                                    2004))))
+                    )
+            )
+        }
+
+        private fun defaultAdresse(): Adresse {
+            return Adresse("Jerpefaret 5C", "1440", "Drøbak", "Norge")
+        }
+
+        private fun defaultBarnetilsynDokumentasjon() = BarnetilsynDokumentasjon(dokumentfelt("barnepassordningFaktura"),
+                dokumentfelt("avtaleBarnepasser"),
+                dokumentfelt("arbeidstid"),
+                dokumentfelt("roterendeArbeidstid"),
+                dokumentfelt("spesielleBehov"))
+
+        data class NavnOgFnr(val navn: String, val fødselsnummer: String)
```

Comment:
Spørsmål: Kan vi flytte denne ut av Builderen?
Så slipper man så lange stier: 
søkerMedBarn.barn.map { NavnOgFnr(it.value.navn.gjeldende().visningsnavn(), it.key) }
vs:
søkerMedBarn.barn.map { TestSøknadBuilder.Builder.NavnOgFnr(it.value.navn.gjeldende().visningsnavn(), it.key) }

Reply:
Ja, det virker jo som en litt "random" data klasse. Hvor skal den puttes ? Trengs den egentlig ?

Reply:
Det er ikke sikkert den trengs lenger. Det var på en måte "første steg" for å legge inn egendefinerte navn/fnr i en testsøknad slik at søknaden henger på greip med pdl-dataene.

Reply:
oki, jeg dro den ut så lenge (utenfor Testsøknad)